### PR TITLE
Implemented first draft of matching algorithm

### PR DESCRIPTION
**Pull request by simoboye**

### CODE REVIEW

**Review by emieboii — APPROVED** [review]

**Review by frederikw03 — APPROVED** [review]
:)

File: BDSA2020.View/Pages/PlacementDescriptionView.razor [hunk 1]
```diff
@@ -2,46 +2,45 @@
...
-            <div style="height: 30px;"></div>
-            <h5>@item.Title</h5>
-            <img src="@item.Thumbnail" alt="">
-                <p class = "description">@item.Description</p>
+    <div class="scroll-container" id="scroll-container">
+        @foreach (var match in matches)
+        {
+            <div class="placement scroll-area">
+                <div class="frame">
+                    <div>Matching score: @match.MatchScore</div>
```
Comment (bjornarjatten) [inline BDSA2020.View/Pages/PlacementDescriptionView.razor:20]: Consider to move it somewhere else

**Review by bjornarjatten — APPROVED** [review]
when-your-pull-request-is-finally-approved-41921383

**Comment by simoboye** [comment]
image

File: BDSA2020.View/Pages/PlacementDescriptionView.razor (hunk 1 above)
Comment (simoboye) [reply BDSA2020.View/Pages/PlacementDescriptionView.razor:20]: Agreed

**Pull request merged**